defaults/main.yml

```diff
@@ -1,14 +1,18 @@
 ---
 
 alpaca_from_source: no
```

Comment:
If this PR still needs to go in, should this line be removed as it appears that you removed the parts from `tasks/deploy.yml` that refer to it?

And if it goes I guess the entire `tasks/compile.yml` could go to?

Reply:
I'll remove what's duplicated in the playbook and let the role define the defaults. The playbook also can now point to the latest build and need not compile from source.

Reply:
Now that I think about it, let me add more from the playbook to this PR.

Reply:
@whikloj @seth-shaw-unlv Good to go now. Once this goes in I'll remove the duplicates in https://github.com/Islandora-Devops/claw-playbook/pull/100